lib/philomena_web/controllers/image/tag_change_controller.ex

```diff
@@ -35,4 +57,17 @@ defmodule PhilomenaWeb.Image.TagChangeController do
 
   defp added_filter(query, _params),
     do: query
+
+  defp log_details(conn, _action, %{image: image, details: details}) do
+    %{
+      body: "Deleted tag change (#{details}) from history",
```

Comment:
maybe also include which image ID is affected in the string?
`Deleted tag change +/-tagname on image 12345 from history.`

Reply:
The reason I didn't do this is because the mod log entry always links to the image page, but I saw now that it's like this on other mod log entries, so I might as well be consistent! Did it!